packages/components/src/components/collapsible/collapsible.css

```diff
@@ -27,11 +27,11 @@
     var(--telekom-color-functional-focus-standard);
 
   /* icon */
-  --spacing-top-icon: var(--telekom-spacing-unit-x1);
+  --icon-size: 24px;
```

Comment:
I wonder if we want to expose this, since I don't remember having this anywhere else. It's not a bad idea, I'm just not sure.

Reply:
removed it, the icon size was set in the tsx file